packages/aws-cdk-lib/aws-servicecatalog/lib/private/product-stack-synthesizer.ts

```diff
@@ -104,6 +104,7 @@ export class ProductStackSynthesizer extends cdk.StackSynthesizer {
         serverSideEncryption: this.serverSideEncryption,
         serverSideEncryptionAwsKmsKeyId: this.serverSideEncryptionAwsKmsKeyId,
         memoryLimit: this.memoryLimit,
+        outputObjectKeys: false,
```

Comment:
Is setting this behaviour to false for this one module the right call? If old users have existing setups that rely on this property being true and the object keys being sent, then this acts as a breaking change for them. We can default this to true to preserve existing behaviour, and if people are running into this issue, they can set this attribute themselves.

Reply:
`ProductStackSynthesizer ` construct is a private construct that is not available for the customers, and also the `bucketDeployment` is also is not accessible by the customers, so I think it is Ok to set this property to false.